src/encrypt.c

```diff
@@ -11,6 +11,8 @@ static void encrypt_none_state_free(void *state);
 static void encrypt_none_encode(char **str, void *key, void *state);
 static void encrypt_none_decode(char **code, void *key, void *state);
 
+static int roll_in_alphabeth(int i, int shift, int alphabeth_size);
```

Comment:
```suggestion
static int roll_in_alphabet(int i, int shift, int alphabet_size);
```
Das ist die Korrekte Schreibweise (auch auf Englisch)